doc/source/data_structures.rst

```diff
@@ -90,18 +89,27 @@ An example using the ``worlds`` GeoDataFrame:
 
 .. ipython:: python
 
+    world = gpd.read_file(gpd.datasets.get_path('naturalearth_lowres'))
+
     world.head()
     #Plot countries
     @savefig world_borders.png width=3in
     world.plot();
 
-Currently, the column named "borders" with country borders is the active
+Currently, the column named "geometry" with country borders is the active
 geometry column:
 
 .. ipython:: python
 
     world.geometry.name
 
+We can also rename this column to "borders":
+
+.. ipython:: python
+
+    world = world.rename(columns={'geometry': 'borders'}).set_geometry('borders')
+    world.geometry.name
+
```

Comment:
@nickeubank I changed this part slightly, because otherwise it is a bit strange that I directly rename the column when I read the file. 
But, I think you wanted to show here that the geometry name does not has to be called 'geometry' ? So that purpose is a bit lost then?